pkg/render/tiers/tiers.go

```diff
@@ -0,0 +1,333 @@
+// Copyright (c) 2022 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tiers
+
+import (
+	"strings"
+
+	"github.com/tigera/operator/pkg/render/kubecontrollers"
+
+	v3 "github.com/tigera/api/pkg/apis/projectcalico/v3"
+	operatorv1 "github.com/tigera/operator/api/v1"
+	"github.com/tigera/operator/pkg/common"
+	"github.com/tigera/operator/pkg/render"
+	rmeta "github.com/tigera/operator/pkg/render/common/meta"
+	"github.com/tigera/operator/pkg/render/common/networkpolicy"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	APIServerPolicyName      = networkpolicy.TigeraComponentPolicyPrefix + "cnx-apiserver-access"
+	ClusterDNSPolicyName     = networkpolicy.TigeraComponentPolicyPrefix + "cluster-dns"
+	KubeControllerPolicyName = networkpolicy.TigeraComponentPolicyPrefix + "kube-controller-access"
+	PacketCapturePolicyName  = networkpolicy.TigeraComponentPolicyPrefix + render.PacketCaptureName
+)
+
+var DNSIngressNamespaceSelector = createDNSIngressNamespaceSelector(
+	render.GuardianNamespace,
+	render.ComplianceNamespace,
+	render.DexNamespace,
+	render.ElasticsearchNamespace,
+	render.LogCollectorNamespace,
+	render.IntrusionDetectionNamespace,
+	render.KibanaNamespace,
+	render.ManagerNamespace,
+	common.TigeraPrometheusNamespace,
+	"tigera-skraper",
```

Comment:
How should we handle this cloud-specific namespace? I adapted this policy from the cloud install as requested, but wasn't sure we should handle this dangling namespace

Reply:
In general, I am a bit confused by the original CC policy, but that could be due to a lack of product/NetworkPolicy knowledge

Why would a managed cluster select for management namespaces? My instinct is that the list of namespaces here should adapt to the cluster type, and that the skraper namespace shouldn't be here at all